src/main/kotlin/com/openlattice/shuttle/control/Integration.kt

```diff
@@ -1,23 +1,67 @@
 package com.openlattice.shuttle.control
 
+import com.fasterxml.jackson.annotation.JsonProperty
 import com.openlattice.client.RetrofitFactory
+import com.openlattice.client.serialization.SerializationConstants
 import com.openlattice.data.integration.StorageDestination
+import com.openlattice.mapstores.TestDataFactory
 import com.openlattice.shuttle.Flight
 import java.util.*
 
 /**
  *
  * @author Matthew Tamayo-Rios &lt;matthew@openlattice.com&gt;
+ *
+ * Represents a data integration, including all fields required to run the integration
+ *
+ * @param environment the retrofit environment (e.g. production, local)
+ * @param defaultStorage ?????
```

Comment:
wat

Reply:
This was a parameter that was in the Integration class before I started working on it and I still don't know what it's purpose is. The "?????" will need to be replaced.

Reply:
talked to matthew and we're getting rid of it :)